### ISSUE TITLE
Where is document for 0.9.x?

### ISSUE THREAD

**Issue by sonots** [embulk/embulk#1112]
https://www.embulk.org/docs/

![image](https://user-images.githubusercontent.com/2290461/53693650-d19cd700-3de6-11e9-9d7f-8bb539e4fc3c.png)

Or, is this correctly 0.9?

**Comment by hiroyuki-sato** [comment]
Related #945
I heard that the document needs to update totally.

**Comment by dmikurube** [comment]
Yeah, we wanted to update the entire document, but it left for a long time...

Just merging #946 could be fine for now, but let me consider.
